Adding lowtown, making bog lower, adding walls, moving gatehouse, adding mayor's house, moving garrison

NikolayByondPlayer opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->

## About The Pull Request

Major town changes,

1) Bastion moved to town, town has walls
![image](https://github.com/user-attachments/assets/87d7cfc4-b19f-41b3-84f3-f7c188407596)

2) Added a few houses and moved them closer to make a basic lowtown for shunned races. Basic stuff and will be improved further
![image](https://github.com/user-attachments/assets/989837cc-4b34-407f-b482-b55c740a0950)

3) Added mayor's house and one new shop and a small square
![image](https://github.com/user-attachments/assets/c9ad5202-cd36-4eba-95c1-4fd3a836b369)

4) Added corridor to docks
![image](https://github.com/user-attachments/assets/983b35fd-b1b3-4c44-b7f6-fc2fe3604d6d)

5) Moved bog one z level down

6) Moved mines closer
![image](https://github.com/user-attachments/assets/2a15cba5-0e0f-4920-8c72-3b50c33966a6)

7) Expansion of sewers, basic stuff will be improved further
![image](https://github.com/user-attachments/assets/2ab277fb-0dfa-4c36-bddf-ae3b085f5a8a)

8) Added some open spaces that look into the sewer from town
![image](https://github.com/user-attachments/assets/2f99c4f7-e585-446c-aeca-f80b65da7368)

9) Moved Soilers and guild
![image](https://github.com/user-attachments/assets/f6b8c6a9-3d8a-4e94-832a-e16b071c06de)

10) Various small fixes

11) Made garrison smaller
![image](https://github.com/user-attachments/assets/20e7d391-e66d-48a8-8a70-3b36e2c1c9d6)

12) Made school less giant 
![image](https://github.com/user-attachments/assets/c8f3333b-aec1-4fc0-a2a3-25acdeb233ba)
![Uploading image.png…]()


## Why It's Good For The Game
Stuff for town that need to be built opon and expanded in future, various QoL, more dark streets

NikolayByondPlayer commented:
Added description

Xaytan commented:
I hope lowtown gets made into a proper slum. I want to live that favella lifestyle.

NikolayByondPlayer commented:
> I hope lowtown gets made into a proper slum. I want to live that favela lifestyle. I want to shantymaxx

Planning to improve low town in future it is a bit basic right now

Faire-Fuzzy commented:
The farm may want to be redone with it be outside the walls now, it it's current state it couldn't stop a determined flea. Not that want farm keep again, but it's laughable the food supply would be so easy to attack. I get that was common in real medieval towns but here it'll just lead to soil folk's play time sucking even harder as six bandits storm up every round for virtually free pickings.

RyszardParys commented:
Just giving more areas for bandits to get free loot... I don't understand the need to keep moving the bog gate/bastion closer and closer to the town area.

radarseas commented:
> Just giving more areas for bandits to get free loot... I don't understand the need to keep moving the bog gate/bastion closer and closer to the town area.

at least this time they'll actually interact with people instead of using their strength to pin and skullcrack 10+ skeles, smelt their shit and randomly show up in dragonscale necklaces.

Kregle commented:
Uh?? WHY REMOVE THE PHYSICIAN'S GARDEN???

K4rlox commented:
this PR removes the keep garden that both the apotheocary, the wizard and the physician is supposed to use to make potions

without even mentioning it in the PR
and replaces it with a fucking
LOW EFFORT BRICK OF WALLS

this change should be removed, this is not good at all

AukariBunn commented:
> LOW EFFORT BRICK OF WALLS
Maintainer (literally a senior developer) is when you dont even care to replace the windows with walls.

Not to mention some other mistakes that can only be done by a newbie in mapping
![image](https://github.com/user-attachments/assets/38751e03-0c26-424d-80b9-08f8770c0048)
![image](https://github.com/user-attachments/assets/ff6906ed-7d77-4731-a7d7-f1b1f960ba6b)
I wont even speak about lots of other illogical decisions and bad mapping as there's no point in arguing with an arrogant fool.

Xaytan commented:
There's a solid argument that the physician's garden is slightly too large, since it definitely has space to mass produce moondust and still grow yourself some crops on the side.

But like. C'mon. A brick? With windows still there? You're better than that.

AukariBunn commented:
> There's a solid argument that the physician's garden is slightly too large, since it definitely has space to mass produce moondust and still grow yourself some crops on the side.

He and Gyran actually wanted to delete it from the game completely to "encourge the court physician to play with the soilsons".

DiscordWizard commented:
Heya, soilson player here. I really like the idea of FINALLY putting the farm somewhere not completely walled and set to a side of the map, as that absolutely destroyed any interaction chances I could ever have and stopped literally no one from breaking in if they really wanted to.

Mapping issues aside, all I want to comment on is to please return the keep farm for the alchemist. I already have my hands full making food for everyone in the city and myself, and the last thing I want is even more tedious work growing alchemical reagents. It's not like it will even stop physicians, who can just commandeer the keep's dirt or lawn for it. It just makes their job more tedious for no apparent gain. It'd be nice to at least have an explanation for some of these changes, see why they are good for the game.

Kregle commented:
> There's a solid argument that the physician's garden is slightly too large, since it definitely has space to mass produce moondust and still grow yourself some crops on the side.
> 
> But like. C'mon. A brick? With windows still there? You're better than that.

No physician had the time to grow food, grow alchemy ingredients, and still tend to folks.

DiscordWizard commented:
> > There's a solid argument that the physician's garden is slightly too large, since it definitely has space to mass produce moondust and still grow yourself some crops on the side.
> 
> He and Gyran actually wanted to delete it from the game completely to "encourge the court physician to play with the soilsons".

Soilson here. I will cut the throat off those physicians. If you want to encourage soilson interaction, just put our farm somewhere open by the main road and force us to sell food to people in person. The stockpile is a terrible rp killer, my one excuse to talk to people is produce related but the stockpile makes it so I just dump grain and suddenly don't have to do anything anymore.

Xaytan commented:
> > There's a solid argument that the physician's garden is slightly too large, since it definitely has space to mass produce moondust and still grow yourself some crops on the side.
> > But like. C'mon. A brick? With windows still there? You're better than that.
> 
> No physician had the time to grow food, grow alchemy ingredients, and still tend to folks.

Do physicians tend folks? I thought they just locked themselves in their goon cave and made drugs all round.

AukariBunn commented:
> No physician had the time to grow food, grow alchemy ingredients, and still tend to folks.

And they believe all physicians will have time to go all the way to the farm to ask the soilsons to grow them faiv billion westleaches

Xaytan commented:
> > No physician had the time to grow food, grow alchemy ingredients, and still tend to folks.
> 
> And they believe all physicians will have time to go all the way to the farm to ask the soilsons to grow them faiv billion westleaches

Ever tried to get Soilsons to grow crops you ask for?

They're all either eaten by goblins, comatose, ignore you, or tell you to fuck off. I just wanted some poison berries for arrows..........

Dilligentdog commented:
I like the slums, i don't like the stealth removal of stuff. 
Firstly removing a entire Z-level makes the bog even more barren to go around. You can no longer loot one of the few dungeons left.
Secondly you removed the entire western portion of outside the town, just lazily replacing it with a stone wall
Thirdly you deleted the court physicians garden to "Encourage gameplay with the soilsons" who NEVER exist, we get at most 1 soilson a round and that's a rare occurrence

Kregle commented:
I'm curious as to how putting the Bastion gate between the Physician and the Soilsons is going to improve the interaction compared to what it was before.

K4rlox commented:
update here

after feedback from players
it seems on https://github.com/Rotwood-Vale/Ratwood-Keep/pull/2062/commits/704ee46a8e42e4b5101079978fbb9df199f4a092
they readded the garden

DeerJesus commented:
https://cdn.discordapp.com/attachments/1281329787746648117/1322983665030004736/image-165.png?ex=6772dc0d&is=67718a8d&hm=32ca75bca1d0d77941632f0a0bb91944b116c9de53865e316815e0440ed9f303& dude you removed the entire west side of town. are you dumb?

DeerJesus commented:
FOUR wide walls covering the ENTIRE town

EIGHT wide moat as well

Is this a joke?

RyszardParys commented:
This is kinda ass, again, but it's funny to see the entire town locked out of the actual town by the bastion gates if they're left untended or no one spawns as a vanguard hahah. Why would latejoiners get to enjoy the game, am I right??

K4rlox commented:
> This is kinda ass, again, but it's funny to see the entire town locked out of the actual town by the bastion gates if they're left untended or no one spawns as a vanguard hahah. Why would latejoiners get to enjoy the game, am I right?? EVEN knights couldnt enter the town.

![image](https://github.com/user-attachments/assets/622c6bc7-7e9a-4f29-9c7e-1d27b9b09e28)

K4rlox commented:
> This is kinda ass, again, but it's funny to see the entire town locked out of the actual town by the bastion gates if they're left untended or no one spawns as a vanguard hahah. Why would latejoiners get to enjoy the game, am I right?? EVEN knights couldnt enter the town.

i was gonna send a screenshot here but i realized i didnt screnshot it

RyszardParys commented:
> > This is kinda ass, again, but it's funny to see the entire town locked out of the actual town by the bastion gates if they're left untended or no one spawns as a vanguard hahah. Why would latejoiners get to enjoy the game, am I right?? EVEN knights couldnt enter the town.
> 
> i was gonna send a screenshot here but i realized i didnt screnshot it

Amazing. Can't wait to get the old rockhill back already. :)

K4rlox commented:
> > > This is kinda ass, again, but it's funny to see the entire town locked out of the actual town by the bastion gates if they're left untended or no one spawns as a vanguard hahah. Why would latejoiners get to enjoy the game, am I right?? EVEN knights couldnt enter the town.
> > 
> > 
> > i was gonna send a screenshot here but i realized i didnt screnshot it
> 
> Amazing. Can't wait to get the old rockhill back already. :)

by the way the screenshot was supposed to be like 15 people including a knight and a steward waiting at the gate because there isnt anybody inside

Slavovski commented:
I do apologise for the following rant, I just wish the server to actually be /good/, not /passable/. Staff, coders, mapper (and the non-existent lore workers who don't do anything), have a to-do list. And stick to it, firmly. 
[------------]
 Server is heading to a "RenaissancePunk" Aesthetic? Cool, great even. Then don't add outfits and sprites that are victorian era and industrial revolution more than actual Renaissance. INFERNAL BARON made them fucking great, cash well spent and props to him, just don't lie to folks about what it /is/. And when folks complain, be honest with them and don't just say "Temporary change".

That was said for the 
A) Bastion-Gate, the changed outfits to a funeral cloak for weeks, if not easily over a full month before an actual sprite was given for it. The bastion was just a dumb change since it's inception, no one wanted it, and a change that keeps getting worse the longer it remains. ROCKHILL FEARS THE COLOUR GREEN. 
B) Mercenary becoming /all/ 3 classes of mercenary. 
C) Removal/remaining of certain roles for literally no reason given, such as Ballif to Marshal for instance.
D) Guns. Still very much a contested choice whether they are good, or bad. Most don't think beyond 'Hah, boomstick'. No concrete explanation given for it's existence either. Are they imported from Grezelhoft? Etrusca? Lavestine Outlaws within rockhill, storing guns in the towner rooms?  
I could count 5 other things, but I digress.
[------------]
Currently, many crucial roles such as smith, merchants and others where the changes were just made with poor design in mind, and poorer execution with the changes. Hell, smithing was bugged for a full two weeks before anything was done to fix it
A solution I bring is... 
[--]
Returning of the Goldface for the merchant, with it dispensing the shipping order directly, upwards of Five items, as that's how the docking ship-trade currently works. Guild tax? doesn't effect the merchant, just rename it to 'Docking fee' or 'Ship fee' and let that be the margin for how much someone needs in order to /import/
Sidenote: Round-start make pipes worthless. I see to many woodcutters heading to the dock, selling 20 pipes and gaining 300 mammon in under 30 min of work. Perhaps add a Peddler which can be changed by the merchant /or guild/ to see what is worth something, swords, armour, potions ect. Wooden items /excluding half-full or full barrels of ale and wine/ shouldn't be equal worth WITHIN rockhill, as they are outside it, makes little sense. I might code that myself. 
[--]
Start the smiths with some extra loot to sell/smelt, some swords, chainmail, a helm or two, random stuff... perhaps 6-8 total? Just so they have /something/ to start off with rather than being told by a Squire they want full-plate for 20 mammon, under 5 min into a round, and they have tied to the steward or miner, which no one plays [miner].  

Misc changes 
VL should be disabled during lowpop. Improve beggars stats-wise. Grenzelhoft Kommandat? Migration role, similar to Zybantines. 
 
Sorry if it sounds sour, not my intention. I thank the staff for the hard work they're doing, they just have to get their ducks in a row, when its anything beyond automatically claiming a ticket as IC issue, or 'Inquis it, bruv". Have a great day.

DelineFortune commented:
Stealth z level removal, thanks

RyszardParys commented:
Ah also... - The latejoiners actually spawn outside of the gate now, even if they're town roles... Now take this fact and add a vanguard full of shit, and you're removed from interacting with the round! Amazing RP-encouraging PR, where i can't even enter the fucking town to RP because some shitter at the gate says he wont let anyone in.
PLEASE for the love of god place the gates back where they were, so when i want to take up the bafoonery bog guards(vanguards) make me fucking go through, i pick a god damn refugee role. I don't want to be damned to a life outside of the town just because some maggot sitting behind his desk said "no".
Not everyone has time to fluoride stare at their screen and wait for a new round to ACTUALLY play the game! Some people gotta latejoin because they have stuff to do other than playing games. All this PR did was just make me not want to play, if I dont make it before the round starts bruh.

ShiningM00N commented:
Honestly I'm just sad they stealth removed the old mines and secret cave. Nikolay promised to keep them :(.

Gyrandola commented:
> Honestly I'm just sad they stealth removed the old mines and secret cave. Nikolay promised to keep them :(.

The area is likely gonna be reused for the High Town as the city rework progresses.

ShiningM00N commented:
> > Honestly I'm just sad they stealth removed the old mines and secret cave. Nikolay promised to keep them :(.
> 
> The area is likely gonna be reused for the High Town as the city rework progresses.
Honestly I'm just tired since there's so little places were I can be a outdoorsman unbothered or have a super private RP place in peace. There used to be the pond by the soilfort, but then the soilfort and THAT got removed. Then I moved it to the east lake, but then that became a transit area for bandits and goblins. So in my last ditch effort I moved it to the secret cave almost nobody knew about, and now that got removed too.

Before you say "Move to the bog", that's a plainly bad idea considering the bandit traffic along the area that goes on. Idk just makes me sad :(

Faire-Fuzzy commented:
A few points (I had them set out more verbose but lost the draft on mobile):

- The Vanguard as their lore and vibe feels vestigial now and they may as well be folded into the watch or rebranded. They're gameplay/role has significantly worsened from the map changes and they are now mostly a multi-gatekeeper keep gate role. Few play keep keep gate master for good reason. The Vanguard now do not encourage RP they are simple dull chain monkeys.
-
- The loss of the Elder for a less flavorful and additional, top heavy role no one will listen to is sad. The Elder was a wonderful village roll and now that they have have again we remove it for what seems like another in the way role. I understand mayor's existed, but this is more or less a one one city duchy and the Duke will always be a walk away and people will just go to him, not to a another pointless ladder rung. I understand that they may stay but at least add Elder back, their position makes more sense and this pr seems to wish to make a massive deal between the high town and low town so give low town a de facto leader again
- 
- The bog is growing into just a place for constant samey bandits and empty unused, uninteresting space. The loss of z level further reduces anyone's interest in going out there. Bandits have no pop restrictions and have a habit of just becoming murder hobos anyway so settlement is not much of a viable prospect. There is no point in the bog, you could map out half of it and save the server processing power at this juncture.

Finally, I understand wanting to bring people together map wise but that can only do so much, antag rework, more interesting bandits, etc. So many other things may have done more to help the vibe of the game and this has seemingly removed several previously interesting avenues in favor a crammed city that may grow dull with tedium and repetition a lot of it's roles now face.

NikolayByondPlayer commented:
> A few points (I had them set out more verbose but lost the draft on mobile):
> 
> * The Vanguard as their lore and vibe feels vestigial now and they may as well be folded into the watch or rebranded. They're gameplay/role has significantly worsened from the map changes and they are now mostly a multi-gatekeeper keep gate role. Few play keep keep gate master for good reason. The Vanguard now do not encourage RP they are simple dull chain monkeys.
> * The loss of the Elder for a less flavorful and additional, top heavy role no one will listen to is sad. The Elder was a wonderful village roll and now that they have have again we remove it for what seems like another in the way role. I understand mayor's existed, but this is more or less a one one city duchy and the Duke will always be a walk away and people will just go to him, not to a another pointless ladder rung. I understand that they may stay but at least add Elder back, their position makes more sense and this pr seems to wish to make a massive deal between the high town and low town so give low town a de facto leader again
> * The bog is growing into just a place for constant samey bandits and empty unused, uninteresting space. The loss of z level further reduces anyone's interest in going out there. Bandits have no pop restrictions and have a habit of just becoming murder hobos anyway so settlement is not much of a viable prospect. There is no point in the bog, you could map out half of it and save the server processing power at this juncture.
> 
> Finally, I understand wanting to bring people together map wise but that can only do so much, antag rework, more interesting bandits, etc. So many other things may have done more to help the vibe of the game and this has seemingly removed several previously interesting avenues in favor a crammed city that may grow dull with tedium and repetition a lot of it's roles now face.

- Bog wil have big templates also we are not dreamkeep - not aiming to have giant dungeons everywhere
- rip velder
-I would honestly just nuke garrison since vanguard are more or less the same thing

NikolayByondPlayer commented:
> > > Honestly I'm just sad they stealth removed the old mines and secret cave. Nikolay promised to keep them :(.
> > 
> > 
> > The area is likely gonna be reused for the High Town as the city rework progresses.
> 
> Honestly I'm just tired since there's so little places were I can be a outdoorsman unbothered or have a super private RP place in peace. There used to be the pond by the soilfort, but then the soilfort and THAT got removed. Then I moved it to the east lake, but then that became a transit area for bandits and goblins. So in my last ditch effort I moved it to the secret cave almost nobody knew about, and now that got removed too.
> 
> Before you say "Move to the bog", that's a plainly bad idea considering the bandit traffic along the area that goes on. Idk just makes me sad :(

No wilderness in the city, go to grove.
Outside will also be dangerous

NikolayByondPlayer commented:
> Stealth z level removal, thanks

check point 5

NikolayByondPlayer commented:
> Ah also... - The latejoiners actually spawn outside of the gate now, even if they're town roles... Now take this fact and add a vanguard full of shit, and you're removed from interacting with the round! Amazing RP-encouraging PR, where i can't even enter the fucking town to RP because some shitter at the gate says he wont let anyone in. PLEASE for the love of god place the gates back where they were, so when i want to take up the bafoonery bog guards(vanguards) make me fucking go through, i pick a god damn refugee role. I don't want to be damned to a life outside of the town just because some maggot sitting behind his desk said "no". Not everyone has time to fluoride stare at their screen and wait for a new round to ACTUALLY play the game! Some people gotta latejoin because they have stuff to do other than playing games. All this PR did was just make me not want to play, if I dont make it before the round starts bruh.

- Latejoin isn's supposed to be too convenient

NikolayByondPlayer commented:
> I do apologise for the following rant, I just wish the server to actually be /good/, not /passable/. Staff, coders, mapper (and the non-existent lore workers who don't do anything), have a to-do list. And stick to it, firmly. [------------] Server is heading to a "RenaissancePunk" Aesthetic? Cool, great even. Then don't add outfits and sprites that are victorian era and industrial revolution more than actual Renaissance. INFERNAL BARON made them fucking great, cash well spent and props to him, just don't lie to folks about what it /is/. And when folks complain, be honest with them and don't just say "Temporary change".
> 
> That was said for the A) Bastion-Gate, the changed outfits to a funeral cloak for weeks, if not easily over a full month before an actual sprite was given for it. The bastion was just a dumb change since it's inception, no one wanted it, and a change that keeps getting worse the longer it remains. ROCKHILL FEARS THE COLOUR GREEN. B) Mercenary becoming /all/ 3 classes of mercenary. C) Removal/remaining of certain roles for literally no reason given, such as Ballif to Marshal for instance. D) Guns. Still very much a contested choice whether they are good, or bad. Most don't think beyond 'Hah, boomstick'. No concrete explanation given for it's existence either. Are they imported from Grezelhoft? Etrusca? Lavestine Outlaws within rockhill, storing guns in the towner rooms? I could count 5 other things, but I digress. [------------] Currently, many crucial roles such as smith, merchants and others where the changes were just made with poor design in mind, and poorer execution with the changes. Hell, smithing was bugged for a full two weeks before anything was done to fix it A solution I bring is... [--] Returning of the Goldface for the merchant, with it dispensing the shipping order directly, upwards of Five items, as that's how the docking ship-trade currently works. Guild tax? doesn't effect the merchant, just rename it to 'Docking fee' or 'Ship fee' and let that be the margin for how much someone needs in order to /import/ Sidenote: Round-start make pipes worthless. I see to many woodcutters heading to the dock, selling 20 pipes and gaining 300 mammon in under 30 min of work. Perhaps add a Peddler which can be changed by the merchant /or guild/ to see what is worth something, swords, armour, potions ect. Wooden items /excluding half-full or full barrels of ale and wine/ shouldn't be equal worth WITHIN rockhill, as they are outside it, makes little sense. I might code that myself. [--] Start the smiths with some extra loot to sell/smelt, some swords, chainmail, a helm or two, random stuff... perhaps 6-8 total? Just so they have /something/ to start off with rather than being told by a Squire they want full-plate for 20 mammon, under 5 min into a round, and they have tied to the steward or miner, which no one plays [miner].
> 
> Misc changes VL should be disabled during lowpop. Improve beggars stats-wise. Grenzelhoft Kommandat? Migration role, similar to Zybantines.
> 
> Sorry if it sounds sour, not my intention. I thank the staff for the hard work they're doing, they just have to get their ducks in a row, when its anything beyond automatically claiming a ticket as IC issue, or 'Inquis it, bruv". Have a great day.

Better write it in dev general - this is wrong place for it

NikolayByondPlayer commented:
> I do apologise for the following rant, I just wish the server to actually be /good/, not /passable/. Staff, coders, mapper (and the non-existent lore workers who don't do anything), have a to-do list. And stick to it, firmly. [------------] Server is heading to a "RenaissancePunk" Aesthetic? Cool, great even. Then don't add outfits and sprites that are victorian era and industrial revolution more than actual Renaissance. INFERNAL BARON made them fucking great, cash well spent and props to him, just don't lie to folks about what it /is/. And when folks complain, be honest with them and don't just say "Temporary change".
> 
> That was said for the A) Bastion-Gate, the changed outfits to a funeral cloak for weeks, if not easily over a full month before an actual sprite was given for it. The bastion was just a dumb change since it's inception, no one wanted it, and a change that keeps getting worse the longer it remains. ROCKHILL FEARS THE COLOUR GREEN. B) Mercenary becoming /all/ 3 classes of mercenary. C) Removal/remaining of certain roles for literally no reason given, such as Ballif to Marshal for instance. D) Guns. Still very much a contested choice whether they are good, or bad. Most don't think beyond 'Hah, boomstick'. No concrete explanation given for it's existence either. Are they imported from Grezelhoft? Etrusca? Lavestine Outlaws within rockhill, storing guns in the towner rooms? I could count 5 other things, but I digress. [------------] Currently, many crucial roles such as smith, merchants and others where the changes were just made with poor design in mind, and poorer execution with the changes. Hell, smithing was bugged for a full two weeks before anything was done to fix it A solution I bring is... [--] Returning of the Goldface for the merchant, with it dispensing the shipping order directly, upwards of Five items, as that's how the docking ship-trade currently works. Guild tax? doesn't effect the merchant, just rename it to 'Docking fee' or 'Ship fee' and let that be the margin for how much someone needs in order to /import/ Sidenote: Round-start make pipes worthless. I see to many woodcutters heading to the dock, selling 20 pipes and gaining 300 mammon in under 30 min of work. Perhaps add a Peddler which can be changed by the merchant /or guild/ to see what is worth something, swords, armour, potions ect. Wooden items /excluding half-full or full barrels of ale and wine/ shouldn't be equal worth WITHIN rockhill, as they are outside it, makes little sense. I might code that myself. [--] Start the smiths with some extra loot to sell/smelt, some swords, chainmail, a helm or two, random stuff... perhaps 6-8 total? Just so they have /something/ to start off with rather than being told by a Squire they want full-plate for 20 mammon, under 5 min into a round, and they have tied to the steward or miner, which no one plays [miner].
> 
> Misc changes VL should be disabled during lowpop. Improve beggars stats-wise. Grenzelhoft Kommandat? Migration role, similar to Zybantines.
> 
> Sorry if it sounds sour, not my intention. I thank the staff for the hard work they're doing, they just have to get their ducks in a row, when its anything beyond automatically claiming a ticket as IC issue, or 'Inquis it, bruv". Have a great day.

Also you can jump and make PRs to move things faster

NikolayByondPlayer commented:
> No west side. Removal of Village Elder, when there's now an actual village. Many keys no longer work. Oh yeah, boys. It's Ratwood time.

many = mayor key......

NikolayByondPlayer commented:
> > > > This is kinda ass, again, but it's funny to see the entire town locked out of the actual town by the bastion gates if they're left untended or no one spawns as a vanguard hahah. Why would latejoiners get to enjoy the game, am I right?? EVEN knights couldnt enter the town.
> > > 
> > > 
> > > i was gonna send a screenshot here but i realized i didnt screnshot it
> > 
> > 
> > Amazing. Can't wait to get the old rockhill back already. :)
> 
> by the way the screenshot was supposed to be like 15 people including a knight and a steward waiting at the gate because there isnt anybody inside

Time for gate party ^^

K4rlox commented:
@NikolayByondPlayer 
![image](https://github.com/user-attachments/assets/6593e670-265b-4b29-aa36-fd7da610d30a)
theres supposed to be a alembic and a mortar on the table here

NikolayByondPlayer commented:
> @NikolayByondPlayer ![image](https://private-user-images.githubusercontent.com/118483925/399265451-6593e670-265b-4b29-aa36-fd7da610d30a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU1NjA1MzIsIm5iZiI6MTczNTU2MDIzMiwicGF0aCI6Ii8xMTg0ODM5MjUvMzk5MjY1NDUxLTY1OTNlNjcwLTI2NWItNGIyOS1hYTM2LWZkN2RhNjEwZDMwYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIzMFQxMjAzNTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lOGM3M2QzYmZiMjA4MTM1ODYxZjIzYzQ2MTVkNzYyMzdhMWFiYzc4MGJiYzNhZWJmNzRiYWI2ZDc3YTEyNWZmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.5jO1a_Tul5SN58kESteT0MQk9qR6KX6fw2bYqdN-xYg) theres supposed to be a alembic and a mortar on the table here

ok

NikolayByondPlayer commented:
Nuked tavern lover area - moved tables up 
![image](https://github.com/user-attachments/assets/95f1fec2-a0e7-48de-978d-27539761cad6)

Githubaccount222 commented:
When you claim a house outside of the gate, the key for the house claimed only works for the front door and not the back door or chests and closets inside said house.

Githubaccount222 commented:
A few more braziers or standing fires in the slums world go a long way too

NikolayByondPlayer commented:
> A few more braziers or standing fires in the slums world go a long way too

can add a tad bit of lights

RyszardParys commented:
> > Ah also... - The latejoiners actually spawn outside of the gate now, even if they're town roles... Now take this fact and add a vanguard full of shit, and you're removed from interacting with the round! Amazing RP-encouraging PR, where i can't even enter the fucking town to RP because some shitter at the gate says he wont let anyone in. PLEASE for the love of god place the gates back where they were, so when i want to take up the bafoonery bog guards(vanguards) make me fucking go through, i pick a god damn refugee role. I don't want to be damned to a life outside of the town just because some maggot sitting behind his desk said "no". Not everyone has time to fluoride stare at their screen and wait for a new round to ACTUALLY play the game! Some people gotta latejoin because they have stuff to do other than playing games. All this PR did was just make me not want to play, if I dont make it before the round starts bruh.
> 
> * Latejoin isn's supposed to be too convenient

Uh huh...

K4rlox commented:
> > Ah also... - The latejoiners actually spawn outside of the gate now, even if they're town roles... Now take this fact and add a vanguard full of shit, and you're removed from interacting with the round! Amazing RP-encouraging PR, where i can't even enter the fucking town to RP because some shitter at the gate says he wont let anyone in. PLEASE for the love of god place the gates back where they were, so when i want to take up the bafoonery bog guards(vanguards) make me fucking go through, i pick a god damn refugee role. I don't want to be damned to a life outside of the town just because some maggot sitting behind his desk said "no". Not everyone has time to fluoride stare at their screen and wait for a new round to ACTUALLY play the game! Some people gotta latejoin because they have stuff to do other than playing games. All this PR did was just make me not want to play, if I dont make it before the round starts bruh.
> 
> * Latejoin isn's supposed to be too convenient

i disagree with this

i believe at least town or guard roles should spawn in the town
like, if nobody joins as a guard roundstart, nobody will be able to open the gate

mix with the fact that you cant strike metal bars
you are technically eternally fucked unless you break the gate

and if you do that
you are even more eternally fucked because there isnt a gate anymore

Xaytan commented:
> > Ah also... - The latejoiners actually spawn outside of the gate now, even if they're town roles... Now take this fact and add a vanguard full of shit, and you're removed from interacting with the round! Amazing RP-encouraging PR, where i can't even enter the fucking town to RP because some shitter at the gate says he wont let anyone in. PLEASE for the love of god place the gates back where they were, so when i want to take up the bafoonery bog guards(vanguards) make me fucking go through, i pick a god damn refugee role. I don't want to be damned to a life outside of the town just because some maggot sitting behind his desk said "no". Not everyone has time to fluoride stare at their screen and wait for a new round to ACTUALLY play the game! Some people gotta latejoin because they have stuff to do other than playing games. All this PR did was just make me not want to play, if I dont make it before the round starts bruh.
> 
> * Latejoin isn's supposed to be too convenient

This is so comically terrible I question your motives. Are you like, a sleeper agent from a rival server or something?

Githubaccount222 commented:
> > When you claim a house outside of the gate, the key for the house claimed only works for the front door and not the back door or chests and closets inside said house.
> 
> Back door is just a simple door

I was talking about the house that has a little balcony. The key doesn't unlock the door to it. It is not a deadbolt.
The key also doesn't unlock any of the three chests or the closet inside.

NikolayByondPlayer commented:
> > > When you claim a house outside of the gate, the key for the house claimed only works for the front door and not the back door or chests and closets inside said house.
> > 
> > 
> > Back door is just a simple door
> 
> I was talking about the house that has a little balcony. The key doesn't unlock the door to it. It is not a deadbolt. The key also doesn't unlock any of the three chests or the closet inside.

Oh let me take a look then

NikolayByondPlayer commented:
> > > When you claim a house outside of the gate, the key for the house claimed only works for the front door and not the back door or chests and closets inside said house.
> > 
> > 
> > Back door is just a simple door
> 
> I was talking about the house that has a little balcony. The key doesn't unlock the door to it. It is not a deadbolt. The key also doesn't unlock any of the three chests or the closet inside.

Fixed!  thanks for pointing out
